denizen_scripts/HeroCraft/mobs/prefixes/attaching.dsc

```diff
@@ -0,0 +1,6 @@
+custom_mob_prefix_attaching:
+  Type: world
+  debug: false
+  events:
+    after entity_flagged:attaching damages entity:
```

Comment:
I don't think this actually does anything

Reply:
honestly idk